extensions/wikia/ImageReview/AbandonedWorkResetter.class.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+class AbandonedWorkResetter extends WikiaModel {
+	/**
+	 * reset state in abandoned work
+	 * note: this is run via a cron script
+	 */
+	public function resetAbandonedWork() {
+		wfProfileIn( __METHOD__ );
+
+		$timeLimit = ( $this->wg->DevelEnvironment ) ? 1 : 3600; // 1 sec
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)